data/json/recipes/other/tool.json

```diff
@@ -216,6 +216,63 @@
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "steel_chunk", 1 ], [ "steel_lump", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "tinderbox",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "survival",
+    "skills_required": [ "fabrication", 2 ],
+    "difficulty": 2,
+    "time": "5 m",
+    "autolearn": true,
+    "book_learn": [ [ "mag_survival", 1 ], [ "textbook_survival", 1 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "can_drink_unsealed", 1 ], [ "can_food_unsealed", 1 ], [ "scrap", 2 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "flint_steel",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "survival",
+    "skills_required": [ "fabrication", 3 ],
+    "difficulty": 4,
+    "time": "10 m",
+    "autolearn": true,
+    "book_learn": [ [ "textbook_survival", 2 ], [ "mag_survival", 3 ] ],
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ [ "sharp_rock", 1 ] ], [ [ "scrap", 1 ] ] ]
```

Comment:
This needs to be rethought, you can't just grab a random chunk of scrap metal and use it as a firestarting steel.
I would recommend evicting it from the PR and re-submitting it as a seperate PR, assuming you can come up with a source of hardened high-carbon steel.